[Pages]: Deploy previews are limited preview-deployments.md (#2880)
cloudflare/cloudflare-docs: products/pages/src/content/platform/preview-deployments.md

Preview deployments allow you to preview new versions of your project without deploying it to production. To view preview deployments, navigate from the **Account Home** > **Pages** and select your project.

> Every time you open a new pull request on your GitHub repository (where the pull request origin is not a fork), Cloudflare Pages will create a unique preview URL, which will stay updated as you continue to push new commits to the branch.

Reviewer: ```suggestion
Every time you open a new pull request on your GitHub repository, Cloudflare Pages will create a unique preview URL, which will stay updated as you continue to push new commits to the branch. This is only true when Pull Requests originate from the repository itself.
```
